### PR TITLE
Replace "DOS box" with "command prompt window" and link to Windows FAQ.

### PR DESCRIPTION
**Pull request by ned-deily**
(Inspired by PR #10703)

### CODE REVIEW

File: Doc/tutorial/interpreter.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ popular alternative location.)
 On Windows machines, the Python installation is usually placed in
 :file:`C:\\Python36`, though you can change this when you're running the
```
Comment (zware) [inline Doc/tutorial/interpreter.rst:27]: This isn't true anymore, is it?

**Review by zware — APPROVED** [review]
The change itself LGTM, but there's more out of date around it.

File: Doc/tutorial/interpreter.rst (hunk 1 above)
Comment (terryjreedy) [reply Doc/tutorial/interpreter.rst:27]: No. Currently, the python dir is put, I believe, in Program Files or Program Files (x86) for all users install or somewhere hidden in Appdate for user install.  I am not sure what to substitute.

Comment (ned-deily) [reply Doc/tutorial/interpreter.rst:27]: I don't have the Windows experience to suggest a better wording so I'll leave it to someone else to write a PR to address that.

**Pull request merged**